mobile/src/bundles/common/components/input/styles.ts

```diff
@@ -3,9 +3,19 @@ import { StyleSheet } from 'react-native';
 import { Color } from '~/bundles/common/enums/enums';
 
 const styles = StyleSheet.create({
+    marker: {
+        backgroundColor: Color.INPUT,
+        borderTopLeftRadius: 5,
+        borderBottomLeftRadius: 5,
+    },
     input: {
         borderWidth: 1,
-        borderColor: '#7e7e7e',
+        borderColor: Color.INPUT,
+        borderTopRightRadius: 5,
+        borderBottomRightRadius: 5,
+    },
```

Comment:
Does this style work correctly if there is no marker?

Reply:
Yes, because in <TextInput> I use 
`!marker && globalStyles.borderRadius5,`
for this